src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/services/OffenceCodeLookupService.kt

```diff
@@ -0,0 +1,179 @@
+
+    return OffenceCodeLookupBuilder()
+      .code(0).nomisCodes("1A", "1B").notCommittedOnOwnNomisCodes("25D").paragraph(paragraph1)
+      .code(1).nomisCodes("1A", "1J").notCommittedOnOwnNomisCodes("25F").paragraph(paragraph1)
+      .code(2).nomisCodes("1A", "1N").notCommittedOnOwnNomisCodes("25G").paragraph(paragraph1)
+      .code(3).nomisCodes("1A", "1F").notCommittedOnOwnNomisCodes("25E").paragraph(paragraph1)
+      .code(4).nomisCodes("1B").notCommittedOnOwnNomisCodes("25D").paragraph(paragraph1)
+      .code(5).nomisCodes("1J").notCommittedOnOwnNomisCodes("25F").paragraph(paragraph1)
+      .code(6).nomisCodes("1N").notCommittedOnOwnNomisCodes("25G").paragraph(paragraph1)
+      .code(7).nomisCodes("1F").notCommittedOnOwnNomisCodes("25E").paragraph(paragraph1)
+      .code(8).nomisCodes("4").notCommittedOnOwnNomisCodes("4").paragraph(paragraph4)
+      .code(9).nomisCodes("5").notCommittedOnOwnNomisCodes("25Q").paragraph(paragraph5)
+      // Empty line in spreadsheet
+      .code(11).nomisCodes("7").notCommittedOnOwnNomisCodes("25O").paragraph(paragraph7)
+      .code(12).nomisCodes("7").notCommittedOnOwnNomisCodes("25O").paragraph(paragraph7)
+      .code(13).nomisCodes("8D").notCommittedOnOwnNomisCodes("25AI").paragraph(paragraph8)
+      .code(14).nomisCodes("8E").notCommittedOnOwnNomisCodes("25AJ").paragraph(paragraph8)
+      // Empty line in spreadsheet
+      .code(16).nomisCodes("12").notCommittedOnOwnNomisCodes("25AM").paragraph(paragraph12)
+      .code(17).nomisCodes("12A").notCommittedOnOwnNomisCodes("25AH").paragraph(paragraph12)
+      .code(18).nomisCodes("14").notCommittedOnOwnNomisCodes("25J").paragraph(paragraph14)
+      .code(19).nomisCodes("13B").notCommittedOnOwnNomisCodes("25J").paragraph(paragraph13)
+      .code(20).nomisCodes("15").notCommittedOnOwnNomisCodes("25T").paragraph(paragraph15)
+      // Empty line in spreadsheet
+      .code(21).nomisCodes("24").notCommittedOnOwnNomisCodes("25Y").paragraph(paragraph24)
+      .code(22).nomisCodes("24").notCommittedOnOwnNomisCodes("25Y").paragraph(paragraph24)
+      .code(23).nomisCodes("23AP").notCommittedOnOwnNomisCodes("25V").paragraph(paragraph23)
+      .code(24).nomisCodes("25Z").notCommittedOnOwnNomisCodes("25Z").paragraph(paragraph23)
+      .code(25).nomisCodes("9").notCommittedOnOwnNomisCodes("25U").paragraph(paragraph9)
+      .code(26).nomisCodes("9").notCommittedOnOwnNomisCodes("25U").paragraph(paragraph9)
+      .code(27).nomisCodes("12AQ").notCommittedOnOwnNomisCodes("25X").paragraph(paragraph12)
+      .code(28).nomisCodes("12AQ").notCommittedOnOwnNomisCodes("25X").paragraph(paragraph12)
+      .code(29).nomisCodes("10").notCommittedOnOwnNomisCodes("10").paragraph(paragraph10)
+      .code(30).nomisCodes("11").notCommittedOnOwnNomisCodes("11").paragraph(paragraph11)
+      // Empty line in spreadsheet
```

Comment:
The linter doesn't like empty lines!

Reply:
Yeah - the TS linter has been causing me all sorts of pain....